swarm/storage/mru/request_test.go

```diff
@@ -77,15 +77,19 @@ func TestEncodingDecodingUpdateRequests(t *testing.T) {
 
 	//Put together an unsigned update request that we will serialize to send it to the signer.
 	data := []byte("This hour's update: Swarm 99.0 has been released!")
-	request := &UpdateRequest{
+	request := &Request{
 		SignedResourceUpdate: SignedResourceUpdate{
-			resourceData: resourceData{
-				version:   1,
-				period:    7,
-				multihash: false,
-				data:      data,
-				metaHash:  metaHash,
-				rootAddr:  rootAddr,
+			resourceUpdate: resourceUpdate{
+				updateHeader: updateHeader{
```

Comment:
This seems pretty complex. Is there an advantage to this complexity?

Reply:
This is so the update header serializer is clearly separated from the data, so adding new information to the header is very clear.

It is true that the initialization of low-level tests look more complicated, but at least you know from reading the code what is each thing.

Reply:
> It is true that the initialization of low-level tests look more complicated,

Right, I missed that this is in the test file. It's fine, then.